wg-docs-tools/README.md

```diff
@@ -17,6 +17,16 @@ Oversees externally-focused tooling (e.g. Fiddle, Forge), Electron documentation
 | <img src="https://github.com/nitsakh.png" width=100 alt="@nitsakh">  | Nitish Sakhawalkar [@nitsakh](https://github.com/nitsakh) | Member | PT (San Francisco) |
 | <img src="https://github.com/shiftkey.png" width=100 alt="@shiftkey">  | Brendan Forster [@shiftkey](https://github.com/shiftkey) | Member | AT (Canada) |
 
+## Current Objective and Key Results
+**Objective:**
+
+As a newcomer or experienced app developer, you can use Electron’s tooling and documentation to build your app well.
+
+**Key Results:**
+* Increase adoption of Electron-provided tooling.
+* Decrease cases of documented poor-practices - normalized against the randomness and size of the samples
```

Comment:
```suggestion
* Decrease percentage of documented poor-practices
```

@codebytere does this resolve your question? I think this is a much clearer way of saying most of what was discussed above.